pytorch_toolkit/object_detection/requirements.txt

```diff
@@ -2,5 +2,4 @@
 -r ../../external/mmdetection/requirements.txt
 networkx==2.3
```

Comment:
Is it necessary to have this version of `networkx`? `networkx` is in the list of MO requirements, and if you have it here only for MO, then I'd suggest to remove it.

Reply:
removed